src/main/java/com/cinema/controller/InjectData.java

```diff
@@ -0,0 +1,46 @@
+package com.cinema.controller;
+
+import com.cinema.model.Role;
+import com.cinema.model.User;
+import com.cinema.service.RoleService;
+import com.cinema.service.ShoppingCartService;
+import com.cinema.service.UserService;
+import java.util.ArrayList;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class InjectData {
```

Comment:
better rename class to DataInitializer and use annotation Component instead of Controller